app/components/access_panels/location_item_component.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= tag.tr class: classes, data: { barcode: item.barcode } do %>
   <td class="callnumber">
-    <%= item.callnumber %>
+    <%= callnumber %>
```

Comment:
Is there a reason this can't be just changed to item.truncated_callnumber and remove the 
```
def callnumber item.truncated_callnumber end
```
in app/components/access_panels/location_consolidated_item_component.rb